src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.html

```diff
@@ -8,136 +8,125 @@
        i18n>Configuration</a>
     <ng-template ngbNavContent>
       <div>
-        <cd-alert-panel
-          *ngIf="!rgwModuleStatus"
-          type="info"
-          spacingClass="mb-3"
-          class="d-flex align-items-center"
-          i18n
-          >In order to access the import/export feature, the rgw module must be enabled
-
+        <cd-alert-panel *ngIf="!rgwModuleStatus"
+                        type="info"
+                        spacingClass="mb-3"
+                        class="d-flex align-items-center"
+                        i18n>In order to access the import/export feature, the rgw module must be enabled
           <button class="btn btn-light mx-2"
                   type="button"
-                  (click)="enableRgwModule()">
-            Enable
-          </button>
+                  (click)="enableRgwModule()">Enable</button>
         </cd-alert-panel>
-        <cd-alert-panel
-          *ngIf="restartGatewayMessage"
-          type="warning"
-          spacingClass="mb-3"
-          i18n>Please restart all Ceph Object Gateway instances in all zones to ensure consistent
-          multisite configuration updates.
+        <cd-alert-panel   *ngIf="restartGatewayMessage"
+                          type="warning"
+                          spacingClass="mb-3"
+                          i18n>Please restart all Ceph Object Gateway instances in all zones to ensure consistent multisite configuration updates.
           <a class="text-decoration-underline"
-             routerLink="/services"> Cluster->Services</a>
+             routerLink="/services">
+             Cluster->Services</a>
         </cd-alert-panel>
-        <cd-table-actions
-          class="btn-group mb-4 me-2"
-          [permission]="permission"
-          [selection]="selection"
-          [tableActions]="createTableActions"
-        >
-        </cd-table-actions>
+        <span *ngIf="!showMultisiteReplicationAction">
```

Comment:
why not if;else for better readability
```suggestion
        <span *ngIf="!showMultisiteReplicationAction; else multisiteReplicationActionTpl">
```